idea/testData/refactoring/extractFunction/controlFlow/evaluateExpression/evalWhenExpr.kt.after

```diff
@@ -8,10 +8,8 @@ fun foo(a: Int): Int {
     return i(a, b)
 }
 
-private fun i(a: Int, b: Int): Int {
```

Comment:
You may say that I should leave previous behaviour. But according to https://youtrack.jetbrains.com/issue/KT-6405 for me it seems that we had a bug and new behavior is correct